daffodil-runtime1/src/main/scala/org/apache/daffodil/processors/parsers/SequenceParserBases.scala

```diff
@@ -224,12 +227,22 @@ abstract class OrderedSequenceParserBase(
               pstate.setSuccess()
               isDone = true
             }
+            case (MissingItem | MissingSeparator | FailureUnspecified) if (!isOrdered) => {
+              pstate.setSuccess()
```

Comment:
Add a comment here as to why for unordered these failure conditions are ok, which I think is because in general unordered stuff behaves like occursCountKind parsed, i.e., it parses until something stops it by a failure.

Reply:
Will do.